frontend/class-frontend.php

```diff
@@ -1675,6 +1676,7 @@ protected function is_multiple_terms_query() {
 	 * @param int    $status   Status code to use.
 	 */
 	public function redirect( $location, $status = 302 ) {
+		header( 'X-Redirect-By: Yoast SEO' );
```

Comment:
We should be aware that WordPress 5.0.0 introduces a third argument for setting the x-redirect-by header. I don't know what will happen when two arguments are set twice. But I guess this header will be overwritten.

https://core.trac.wordpress.org/browser/trunk/src/wp-includes/pluggable.php#L1341